packages/perspective-jupyterlab/src/config/plugin.config.js

```diff
@@ -25,11 +25,45 @@ module.exports = {
     plugins: [new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /(en|es|fr)$/), new PerspectivePlugin()],
     module: {
         rules: [
+            {
+                test: /\.less$/,
+                exclude: /themes/,
+                use: [
+                    {
+                        loader: "css-loader"
+                    },
+                    {
+                        loader: "clean-css-loader",
+                        options: {
+                            level: 2
+                        }
+                    },
+                    {
+                        loader: "less-loader"
+                    }
+                ]
+            },
+            {
+                test: /\.(html)$/,
+                loader: "html-loader"
+            },
+            {
+                test: /\.(arrow)$/,
+                loader: "arraybuffer-loader"
+            },
+            {
```

Comment:
The Jupyterlab plugin does not even use babel - it is written in Typescript.  Is it not a red flag that this refactor requires so much extra webpack configuration for e.g. this package?

Reply:
Yup. Removed.